internal/db/schema/migrations/oss/postgres/27/01_disable_terminate_session.up.sql

```diff
@@ -0,0 +1,36 @@
+begin;
+-- Replaces function from 0/51_connection.up.sql
+-- Remove call to terminate_session_if_possible
+drop trigger update_connection_state_on_closed_reason on session_connection;
+drop function update_connection_state_on_closed_reason();
+create function
+    update_connection_state_on_closed_reason()
+    returns trigger
+as $$
+begin
+    if new.closed_reason is not null then
+        -- check to see if there's a closed state already, before inserting a new one.
+        perform from
+            session_connection_state cs
+        where
+                cs.connection_id = new.public_id and
+                cs.state = 'closed';
+        if not found then
+            insert into session_connection_state (connection_id, state)
+            values
+                (new.public_id, 'closed');
+end if;
+end if;
```

Comment:
the indentation of these seems off. they should align with the corresponding `if`